product_docs/docs/pgd/5/reference/functions-internal.mdx

```diff
@@ -223,21 +222,21 @@ Used internally to implement the `bdr.node_slots` view.
 ### `bdr.internal_alter_sequence_set_kind`
 
 PGD internal function used internally for replication of the various function calls.
-
+<! Could this just start with "Used internally" >
 No longer used by the current version of PGD. Exists only for backward
 compatibility during rolling upgrades.
 
 ### `bdr.internal_replication_set_add_table`
 
 PGD internal function used internally for replication of the various function calls.
-
+<! Could this just start with "Used internally" >
```

Comment:
Same query (Start with "used internally"?)

Reply:
A function previously used internally....
(as we expand on in the next line)